k8s/base/juno/juno-db-snapshot.dockerfile

```
@@ -0,0 +1,10 @@
+FROM alpine:3.20
+
+RUN apk update && apk add --no-cache wget tar
```

Hadolint

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)